zio-http/src/main/scala/zio/http/codec/QueryCodecs.scala

```diff
@@ -15,30 +15,36 @@
  */
 
 package zio.http.codec
+import zio.Chunk
 import zio.stacktracer.TracingImplicits.disableAutoTrace
 private[codec] trait QueryCodecs {
   def query(name: String): QueryCodec[String] =
-    HttpCodec.Query(name, TextCodec.string)
+    HttpCodec.MonoQuery(name, TextCodec.string)
 
   def queryBool(name: String): QueryCodec[Boolean] =
-    HttpCodec.Query(name, TextCodec.boolean)
+    HttpCodec.MonoQuery(name, TextCodec.boolean)
 
   def queryInt(name: String): QueryCodec[Int] =
-    HttpCodec.Query(name, TextCodec.int)
+    HttpCodec.MonoQuery(name, TextCodec.int)
 
   def queryAs[A](name: String)(implicit codec: TextCodec[A]): QueryCodec[A] =
-    HttpCodec.Query(name, codec)
+    HttpCodec.MonoQuery(name, codec)
+
+  def queries[I](name: String)(implicit codec: TextCodec[I]): QueryCodec[Chunk[I]] =
+    HttpCodec.MultiQuery(name, codec)
 
   def paramStr(name: String): QueryCodec[String] =
-    HttpCodec.Query(name, TextCodec.string)
+    HttpCodec.MonoQuery(name, TextCodec.string)
```

Comment:
@jdegoes I think we should delete one group of constructors. Either query or param not both.

Reply:
@987Nabil I think so. I just thought one of groups needed for backward compatibility.

Reply:
@eshu we are in RC versions. There is no compat. guarantees. This is the time to remove one of them. Now or never basically ;)

Reply:
@987Nabil I think `param` should be better, but in this case it will be correct to change object `Query` name too

Reply:
Yes, we should be consistent with naming.

Reply:
@987Nabil @jdegoes So what should I drop - queries or params? If I drop queries, I need to rename Query objects to Param, correct?

Actually I prefer to drop queries and rename, but you are owners of the project, the decision should be yours.

Reply:
I think param is to general. A header can also be seens as a param. I prefer query. And it is consistent with the name query codec.

Reply:
Ok